docs/user-guide/street-view.md

```diff
@@ -0,0 +1,23 @@
+# Street View
+*******************
+
+The **Street View** is an extension that can be included in the third step of the application context wizard to allow the end user to browse data from Google street view. The <img src="../img/button/street-view-button.jpg" class="ms-docbutton"/> button, present in *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, provides to the user the pop-up to navigate the map with Google street view. 
+
+<img src="../img/street-view/streetview-popup.jpg" class="ms-docimage" width="500px"/>
+
+Once the *Street View* pop-up opens, the streets on the map are highlighted and the user can select one on the map.
+
+<img src="../img/street-view/add-street.gif" class="ms-docimage"/>
+
+At this point the road is visible in the pop-up and the editor can:
```

Comment:
Clicking on a street in map the the tool's window visualize the Street View and the user can navigate it as usual.